provider/pkg/await/daemonset_test.go

```diff
@@ -320,20 +320,26 @@ func TestAwaitDaemonSetDelete(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt
```

Comment:
BTW I learned that loop variables are now copied as of go 1.22 and you don't need this line anymore.